rehive/api/client.py

```diff
@@ -124,14 +124,15 @@ def _request(self,
                 if result.status_code == 500:
                     raise APIException('Internal server error: ' + url, result.status_code)
                 try:
-                    error_data = result.json()
-                    raise APIException(error_data.get(
-                            'message', 'General error'),
-                            result.status_code, error_data)
+                    raise APIException(
```

Comment:
This is incorrect. You did not have to change anything inside the `try` part of the exception handling.

You are now not doing the JSON decode anymore... which means that errors will never be parsed if they are in JSON. Leave line 127-130 as they were before.